spec/compiler/parser/parser_spec.cr

```diff
@@ -1949,8 +2055,14 @@ module Crystal
     it_parses "foo 1.bar do\nend", Call.new("foo", args: [Call.new(1.int32, "bar")] of ASTNode, block: Block.new)
     it_parses "return 1.bar do\nend", Return.new(Call.new(1.int32, "bar", block: Block.new))
 
-    %w(begin nil true false yield with abstract def macro require case if unless include extend class struct module enum while
-      until return next break lib fun alias pointerof sizeof instance_sizeof offsetof typeof private protected asm end do else elsif when rescue ensure).each do |keyword|
+    %w(
+      begin nil true false yield with abstract
+      def macro require case select if unless include
+      extend class struct module enum while until return
+      next break lib fun alias pointerof sizeof
+      instance_sizeof offsetof typeof private protected asm
+      end self in do else elsif when rescue ensure
```

Comment:
`in` ain't keyword, innit?

```suggestion
      end self do else elsif when rescue ensure
```

Reply:
It is bruv https://crystal-lang.org/reference/1.15/syntax_and_semantics/case.html#exhaustive-case

Reply:
True story, my bad.